e2e/chat.spec.ts

```diff
@@ -109,12 +109,12 @@ describe('chat', () => {
     //  Confirm the tx
     await chat.submitMessage('Yes');
 
-    //  In progress
-    await expect(page.getByTestId('in-progress-tx-display')).toBeVisible();
-
     const metamaskPopup = await metamaskPopupPromise;
     await confirmMetamaskTransaction(metamaskPopup);
 
+    //  In progress
+    await expect(page.getByTestId('in-progress-tx-display')).toBeVisible();
+
```

Comment:
This had to change in the automated test because when they click "Yes", we still need to connect their wallet to internal testnet, so they haven't passed the validation step yet, so the in progress component isn't displaying yet.

Makes me think we could possibly repurpose this card to show "Confirm connection in Metamask" if a user is here for the first time and hasn't given permission to connect.

Reply:
https://github.com/user-attachments/assets/e497bd10-bf67-4dde-85f0-bccbca727f34